.github/dependabot.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "03:00"
+    timezone: Europe/London
+  open-pull-requests-limit: 10
+  reviewers:
+  - kmahern
+  - mabely
```

Comment:
@mabely I think we should just have you and me here for the time being for reviewing the ruby/rails dependabot prs and have @malcomvonmj as reviewer for reviewing the frontend the npm/webpacker prs (instead of @naseberry).